apps/docs/content/components/Button.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Button
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=2931%3A30104'
+source: https://github.com/primer/brand/blob/main/packages/react/src/Button/Button.tsx
+storybook: '/brand/storybook/?path=/story/components-button--playground'
+description: Use Button for the main actions on a page or form.
+---
+
+import {ButtonVariants, ButtonSizes} from '@primer/react-brand'
+import {Box as Container} from '@primer/react'
+import {SearchIcon} from '@primer/octicons-react'
+import {PropTableValues} from '../../src/components'
+
+```js
+import {Button} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+This is the default variant for the Button component. It corresponds to the `secondary` variant appearance.
+
+```jsx live
+<Button>Default</Button>
+```
+
+### Primary
+
+The `primary` variant of Button indicates that the action is of high priority, and should be used sparingly.
+
+```jsx live
+<Button variant="primary">Primary</Button>
+```
+
+### Subtle
+
+The `subtle` variant of Button indicates that the action is a low priority one.
+
+```jsx live
+<Button variant="subtle">Primary</Button>
+```
+
+### Sizes
+
+```jsx live
+<Stack direction="vertical" alignItems="flex-start">
+  <Button size="medium">Register now</Button>
+  <Button size="large">Register now</Button>
+</Stack>
+```
+
+### Removing the arrow
+
+The animated arrow indicator can be removed using the `hasArrow` prop.
+
+```jsx live
+<Button hasArrow={false}>No arrow</Button>
+```
+
+### Appending an icon
+
+We can place an icon inside the Button in either the leading or the trailing position to enhance the visual context. It is recommended to use an [Octicon](https://primer.style/octicons) here.
+
+```jsx live
+<Stack direction="vertical" alignItems="flex-start">
+  <Button leadingVisual={SearchIcon}>Search</Button>
+  <Button trailingVisual={SearchIcon}>Search</Button>
+  <Button leadingVisual={SearchIcon} trailingVisual={CheckIcon}>
+    Search
+  </Button>
+</Stack>
```

Comment:
This examples could be misleading using `search` for the trailingVisual.

 ```suggestion
<Stack direction="vertical" alignItems="flex-start">
  <Button leadingVisual={SearchIcon}>Search</Button>
  <Button trailingVisual={ChevronDown}>Select</Button>
  <Button leadingVisual={Filter} trailingVisual={ChevronDown}>
    Filter
  </Button>
</Stack>
```

Reply:
Great point. Thanks @josepmartins 👍. Changed now. I'll make same change in the stories for consistency.